products/cloudrun/terraform.yaml

```diff
@@ -152,6 +152,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       spec.template.spec.containerConcurrency: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+      spec.template.spec.timeoutSeconds: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Is the default a known value, or might it change depending on how a user configures their service? If we know what it's going to be, it makes more sense to hardcode the default (https://github.com/terraform-providers/terraform-provider-google/wiki/Developer-Best-Practices#default-values)

Reply:
it's depends for fully managed or Cloud Run for Anthos.
From documentations https://cloud.google.com/run/docs/configuring/request-timeout#command-line it's 5 minutes for  fully managed(but I think it's 15 minutes from my last test).
I think better use default from API, because it's related to user billing because fully managed cloud run  users pay for time of request.

Reply:
Makes sense, thanks for the explanation!